src/main/java/org/fundacionjala/pivotal/core/api/APICommons.java

```diff
@@ -47,24 +57,36 @@ public static String getElement(final Response res, final String element) {
      * @param response the response from request.
      */
     public static void saveResponse(final String variableName, final Response response) {
-        requestResponse = new HashMap<>();
+        //requestResponse = new HashMap<>();
         requestResponse.put(variableName, response);
     }
 
     /**
      * Method to get map with response.
      * @return the map.
      */
-    public static Map getRequestResponse() {
+    public static Map<String, Response> getRequestResponse() {
         return requestResponse;
     }
 
     /**
      * Method to build endpoint.
-     * @param res the response to a request.
+     * @param  endPoint response to a request.
      */
-    public static void buildEndPoint(final Response res) {
+   /* public static void buildEndPoint(final Response res) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)